umi inspect 命令有问题

D:\vue react\iatk-master>umi inspect
module.js:549
throw err;
^

Error: Cannot find module 'af-webpack/webpack-chain'
at Function.Module._resolveFilename (module.js:547:15)
at Function.Module._load (module.js:474:25)
at Module.require (module.js:596:17)
at require (internal/module.js:11:18)
at api.registerCommand (D:\vue react\iatk-master\node_modules\umi-build-dev\lib\plugins\commands\inspect\index.js:25:26)
at Service.run (D:\vue react\iatk-master\node_modules\umi-build-dev\lib\Service.js:346:12)
at Object. (D:\vue react\iatk-master\node_modules\umi\lib\scripts\inspect.js:19:47)
at Module._compile (module.js:652:30)
at Object.Module._extensions..js (module.js:663:10)
at Module.load (module.js:565:32)

--- a/packages/af-webpack/package.json
+++ b/packages/af-webpack/package.json
@@ -99,6 +99,7 @@
     "getUserConfigPlugins.js",
     "react-dev-utils.js",
     "registerBabel.js",
+    "webpack-chain.js",
     "webpack.js"
   ],
   "umiTools": {

